### PR TITLE
Fix malformed attribute for specifying a custom repo recipe

### LINKED ISSUE #38: Custom repository "option" non-functional due to malformed attribute
While well-intentioned, the "passthrough" attribute for a custom repository is non-functional due to attribute misnaming:

https://github.com/chef-cookbooks/supermarket-omnibus-cookbook/blob/7fdad3a40a8d93bbddcbd884d9d88aca6b822d79/libraries/supermarket_server.rb#L81

Should be:
```
node.set['chef-ingredient']['custom-repo-recipe'] = node['supermarket_omnibus']['custom_repo_recipe']
```

_(--hyphens-- not __underscores__)

### PR DESCRIPTION
**Pull request by yzl**
Fixes #38.

cc @robbkidd @nellshamrell

### CODE REVIEW

**Comment by jaymalasinha** [comment]
👍

File: spec/unit/test_custom_repo_setup_spec.rb [hunk 1]
```diff
@@ -4,17 +4,13 @@
...
 describe 'test::custom_repo_setup' do
   context 'When a custom recipe is specified, the custom recipe should be used' do
     let(:chef_run) do
       runner = ChefSpec::SoloRunner.new(platform: 'redhat', version: '6.5', step_into: %w(chef_ingredient supermarket_server)) do |node|
-        node.set['supermarket_omnibus']['custom_repo_recipe'] = 'custom_repo::custom_repo_recipe'
-        node.set['supermarket_omnibus']['chef_server_url']    = 'https://chefserver.mycorp.com'
+        node.set['supermarket_omnibus']['chef_server_url'] = 'https://chefserver.mycorp.com'
         node.set['supermarket_omnibus']['chef_oauth2_app_id'] = 'blahblah'
-        node.set['supermarket_omnibus']['chef_oauth2_secret'] = 'bob_loblaw'
+        node.set['supermarket_omnibus']['chef_oauth2_secret'] = 'bob_lawblaw'
```
Comment (irvingpop) [inline spec/unit/test_custom_repo_setup_spec.rb]: ❤️

**Comment by irvingpop** [comment]
👍 :shipit:

**Comment by robbkidd** [comment]
👍

**Pull request merged**